napari/utils/settings/_defaults.py

```diff
@@ -57,6 +73,7 @@ class ApplicationSettings(BaseSettings, EventedModel):
     window_state: str = None
     window_statusbar: bool = True
     preferences_size: Tuple[int, int] = None
+    # TODO: Might be breaking preferences?
```

Comment:
yeah... I just played with this and see that adding these enums don't work.  They are there in the 'definitions' section of the model json schema... 

<details>

```json
{
  "title": "Application settings",
  "description": "Main application settings.",
  "type": "object",
  "additionalProperties": False,
  "definitions": {
    "NotificationSeverity": {
      "title": "NotificationSeverity",
      "description": "Severity levels for the notification dialog.  Along with icons for each.",
      "enum": ["error", "warning", "info", "debug", "none"]
    }
  },
  "properties": {
    "theme": {
      "title": "Theme",
      "description": "Theme selection.",
      "default": "dark",
      "env_names": ["napari_theme"],
      "enum": ["dark", "light"],
      "type": "string"
    },
    "gui_notification_level": {
      "default": "info",
      "env_names": ["napari_gui_notification_level"],
      "allOf": [{ "$ref": "#/definitions/NotificationSeverity" }]
    },
    "console_notification_level": {
      "default": "none",
      "env_names": ["napari_console_notification_level"],
      "allOf": [{ "$ref": "#/definitions/NotificationSeverity" }]
    }
  }
}
```

</details>

but it [doesn't look like `qt_jsonchema_form` supports $refs](https://github.com/agoose77/qt-jsonschema-form#unsupported-validators).  Do you guys have thoughts on how to address that?

Reply:
I'm going to start looking into this and talk with @goanpeca and @Carreau about a plan.  If you have any initial thoughts, @tlambert03 , I'd love to hear them.  :)

Reply:
I don't have any initial thoughts... just wanted to flag that as a potential issue that we will likely come up against quickly.  Any sort of workaround is fine with me.

Reply:
As discussed in meeting we can defer this until after merge and follow up PRs as needed. Thanks for looking in!